app/controllers/api/exercises/scratchpad_page_controller.rb

```diff
@@ -0,0 +1,34 @@
+module API
+  module Exercises
+    class ScratchpadPageController < BaseController
+      before_action :use_exercise
+
+      def show
+        page = ScratchpadPage.find_by(about: @exercise, author: current_user)
+
+        render json: SerializeScratchpadPage.(page)
+      end
+
+      def create
+        page = ScratchpadPage.create!(scratchpad_page_params)
```

Comment:
This should be `update` and `create_or_find_by`. We don't need separate methods for creating/updating. The public user can just always presume there is a page for every possible exercise and use `patch` throughout. That makes it much simpler for the consumer.